### PR TITLE
hidden password upgrade notice and removal of change password/new wallet buttons

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Comment by douglasbakkum** [comment]
ack

File: src/qt/ui/settingsdialog.ui [hunk 1]
```diff
@@ -243,7 +243,7 @@
             </sizepolicy>
            </property>
            <property name="text">
-            <string>As a security measure in compromised situations, use a special password to open a hidden wallet. The hidden wallet can receive and send coins, and it has 2FA mode permanently enabled.</string>
+            <string>As a security measure in compromised situations, use a special password to open a hidden wallet. The hidden wallet can receive and send coins, and it has 2FA mode permanently enabled. NOTE: If you have funds in a hidden wallet created before firmware version v3.0.0, move the funds out before setting a new hidden wallet password.</string>
```
Comment (jonasschnelli) [inline src/qt/ui/settingsdialog.ui:246]: Doesn't this belong more to the release notes?

**Comment by jonasschnelli** [comment]
ACK once the PR title and message is set (PR title and message will be part of the git history)

**Comment by benma** [comment]
@jonasschnelli done, changed the PR title.

**Pull request merged**